rfcs/webtransport_h3_test_server.md

```diff
@@ -0,0 +1,94 @@
+# RFC 85: WebTransport over HTTP/3 Test Server
+
+## Summary
+
+Add a [WebTransport over HTTP/3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3-01) server to wpt. The server is used to test the [WebTransport Web APIs](https://w3c.github.io/webtransport/).
+
+The server will use a new [file name flag](https://web-platform-tests.org/writing-tests/file-names.html) `.wt.h3` (`.wt.h3.py`) for handlers. Handlers are python scripts which are used to respond WebTransport events.
```

Comment:
So, usually the filename flags are used to indicate that the test file itself is served over some particualr protocol. But I assume that webtransport tests will actually be testharness tests. So the `h3` flag in the test filename might be needed so the test can be same origin with the http3 server. But it's unclear why it would be needed in the name of the the `.py` file; that will just be loaded on whichever server the test requests.

Conversely the `.wt` part is presumably so that the server code knows to provide the correct python API (i.e. `handle_event` rather than `main`), so that seems reasonable to include in the name of the python file, but it's not needed in the name of the test itself.

I know the QUIC setup currently uses `.quic` in the name of Python files, but I think that's largely because the Python 3 requirement wasn't standard at the time it was introduced, so running the quic tests introduced an additional dependency that wasn't met in all environments.

Reply:
Thank you for the context. I used `.wt` or `.webtransport` largely because `.quic` is used for QuicTransport. Name flags are removed from the current PR.